infra/dev-personal/main.tf

```diff
@@ -33,119 +33,149 @@ module "psoxy-gcp" {
   ]
 }
 
-module "gmail-connector" {
-  source = "../modules/google-workspace-dwd-connection"
-
-  project_id                   = var.project_id
-  connector_service_account_id = "psoxy-gmail-dwd"
-  display_name                 = "Psoxy Connector - GMail${var.connector_display_name_suffix}"
-  apis_consumed                = [
-    "gmail.googleapis.com"
-  ]
-  oauth_scopes_needed          = [
-    "https://www.googleapis.com/auth/gmail.metadata"
-  ]
-
-  depends_on = [
-    module.psoxy-gcp
-  ]
-}
-
-# setup gmail to auth using a secret (not specific to Cloud Function)
-module "gmail-connector-auth" {
-  source = "../modules/gcp-sa-auth-key-secret-manager"
-
-  secret_project     = var.project_id
-  service_account_id = module.gmail-connector.service_account_id
-
-  # TODO: recommend migrate this to `PSOXY_{{function_name}}_SERVICE_ACCOUNT_KEY`
-  secret_id          = "PSOXY_SERVICE_ACCOUNT_KEY_gmail"
-}
-
-# for local dev, write the key to flat file on your machine (shouldn't do this for prod configs)
-resource "local_file" "gmail-connector-sa-key" {
-  filename = "gmail-connector-sa-key.json"
-  content_base64 = module.gmail-connector-auth.key_value
-}
-
-module "psoxy-gmail" {
-  source = "../modules/gcp-psoxy-cloud-function"
-
-  project_id            = var.project_id
-  function_name         = "psoxy-gmail"
-  source_kind           = "gmail"
-  service_account_email = module.gmail-connector.service_account_email
-
-  secret_bindings = {
-    PSOXY_SALT = {
-      secret_name    = module.psoxy-gcp.salt_secret_name
-      version_number = module.psoxy-gcp.salt_secret_version_number
-    },
-    SERVICE_ACCOUNT_KEY = {
-      secret_name    = module.gmail-connector-auth.key_secret_name
-      version_number = module.gmail-connector-auth.key_secret_version_number
+locals {
+  # Google Workspace Sources; add/remove as you wish
+  google_workspace_sources = {
+    # GDirectory connections are a PRE-REQ for gmail, gdrive, and gcal connections. remove only
+    # if you plan to directly connect Directory to worklytics (without proxy). such a scenario is
+    # used for customers who care primarily about pseudonymizing PII of external subjects with whom
+    # they collaborate in GMail/GCal/Gdrive. the Directory does not contain PII of subjects external
+    # to the Google Workspace, so may be directly connected in such scenarios.
+    "gdirectory": {
+      deploy: 1
```

Comment:
this is just the test example. I put it up to date with your recent changes. I added this `deploy: 1` just to test.
Instead of removing the ones you don't want, just mark it as deployable or not.